src/app/components_ngrx/label-selector/label-selector.component.ts

```diff
@@ -210,6 +210,9 @@ export class LabelSelectorComponent implements OnInit {
   }
 
   onAddLabelInput(val) {
-    this.createDisabled = val === '';
+    if (val.trim().length === 0 && val.length) {
+      this.labelnameInput.nativeElement.value = '';
+    }
+    this.createDisabled = val.trim() === '';
```

Comment:
`this.createDisabled = true` can go inside the true. It's repeating condition check here. 
You can have a else block for `this.createDisabled = false`